freqtrade/optimize/hyperopt.py

```diff
@@ -295,6 +296,50 @@ def _format_explanation_string(results, total_epochs) -> str:
                 f"{results['results_explanation']} " +
                 f"Objective: {results['loss']:.5f}")
 
+    @staticmethod
+    def print_result_table(config: dict, results: list, total_epochs: int, highlight_best: bool,
+                           print_colorized: bool) -> None:
+        """
+        Log result table
+        """
+        if not results:
+            return
+
+        trials = json_normalize(results, max_level=1)
+        trials = trials[['is_best', 'current_epoch',
+                         'results_metrics.trade_count', 'results_metrics.avg_profit',
+                         'results_metrics.total_profit', 'results_metrics.profit',
+                         'results_metrics.duration', 'loss']]
+        trials.columns = ['Best', 'Epoch', 'Trades', 'Avg profit', 'Total profit',
+                          'Profit', 'Avg duration', 'Objective']
+
+        trials['Best'] = trials['Best'].apply(lambda x: '*' if x else '')
```

Comment:
I propose to keep the '*' mark for epochs that randomly seed, and use another mark (for example '!' or '+' for best results. Both can be set. The mark for the best results help when no colorization is used.

Reply:
I changed so 'is_initial_point' will set '*' and is_best will set 'Best'
We have space for that so might be more intuitive for new users.